main/src/routes/middleware.js

```diff
@@ -27,54 +34,27 @@ function checkCredentials(req, res, next) {
 }
 
 /**
- * Tries refreshing the credentials if they are invalid.
- * Used as middle ware function
- * @param req - current incoming request
- * @param res - current response to request
- * @param next - handler for next fn
- * @returns {Promise<T | void>|*|void|Response}
+ * Checks validity of request body for POST /webcam
```

Comment:
Do you want to get rid of the req, res, and next for all of them if we erased it here and keep it for the other fns?

Reply:
Oops I was not supposed to delete that thanks for catching that!